test/connectors/CodeListConnectorISpec.scala

```diff
@@ -23,7 +23,7 @@ import java.util.Locale.{ENGLISH, JAPANESE}
 
 import scala.collection.immutable.ListMap
 
-class CodeListConnectorSpec extends ConnectorSpec {
+class CodeListConnectorISpec extends ConnectorISpec {
```

Comment:
This is not really an integration test (no use of wiremock) so I'd leave out the "ISpec" postfix.

Reply:
I'll change then the extended trait.